package.json

```
@@ -11,7 +11,7 @@
     "lint-fix": "eslint --fix .",
     "dev": "docker-compose up --build",
     "dev-client": "npm run --prefix client start",
-    "dev-server": "nodemon --exec 'node --inspect -r ts-node/register' server/src/index.ts",
+    "dev-server": "ts-node-dev --respawn --transpileOnly server/src/index.ts",
```

Just so we have debugging enabled out of the box, how about..
```suggestion
    "dev-server": "ts-node-dev --respawn --transpileOnly --inspect -- server/src/index.ts",
```
See ts-node-dev documentation with regards to supplying flags unknown to it